angular/projects/admin-nrpti/src/app/records/inspections/inspection-nrced-detail/inspection-nrced-detail.component.ts

```diff
@@ -49,6 +50,16 @@ export class InspectionNRCEDDetailComponent extends RecordComponent implements O
     this.isPublished = this.isRecordPublished();
   }
 
+  canPublish(): boolean {
+    const requiredRoles = Constants.FlavourEditRequiredRoles.INSPECTION.LNG;
```

Comment:
Should this be INSPECTION.NRCED?

Reply:
Thanks for the catch! Fixed in https://github.com/bcgov/NRPTI/pull/774/commits/9c859301d4d69ac2b8c19b44540b2e8b3fa610d9